packages/docs/functions-generator.js

```diff
@@ -1,6 +1,35 @@
 const jsdocParser = require('jsdoc3-parser');
+const path = require('path');
+const fse = require('fs-extra');
+const { exec } = require('child_process');
 
-async function parseFile(file) {
+async function parseTSFile(file) {
+    return new Promise((resolve, reject) => {
```

Comment:
I know it's the same pattern as previously used. But I am not a fan of callbacks, could this be solved by using [util.html#utilpromisifyoriginal](https://nodejs.org/api/util.html#utilpromisifyoriginal)? That way we'd fully use `async`/`await` and no need to mess with callbacks and nesting.

Reply:
I mean it could, but then this one function would be entirely different to every other user to generate the docs. 

I can imagine this to be a nice task for one of our upcoming interns to get familiar with the environment.

Reply:
Good point! I've created issue to addess this https://github.com/RedHatInsights/frontend-components/issues/1326